src/composables/transferSendForm.ts

```diff
@@ -92,14 +94,31 @@ export function useTransferSendForm({
     };
   }
 
+  /**
+   * Save form data on local storage to restore it
+   * e.g. after extension closes & opens window to ask camera permission
+   */
+  function saveFormDataOnLocalStorage() {
+    setLocalStorageItem([TRANSFER_SEND_DATA_LOCAL_STORAGE_KEY], formModel.value);
+  }
```

Comment:
Instead of this I suggest to use `useStorageRef` taken from the `src/composables/composablesHelpers.ts`

Reply:
I could not get this to work, and it seems simpler to just use `setLocalStorageItem`